crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -630,6 +636,13 @@ pub struct RenderMeshInstancesCpu(MainEntityHashMap<RenderMeshInstanceCpu>);
 #[derive(Default, Deref, DerefMut)]
 pub struct RenderMeshInstancesGpu(MainEntityHashMap<RenderMeshInstanceGpu>);
 
+/// FIXME: The fact that we need this is ugly.
```

Comment:
This comment would be more useful if you could also say why we need it, so we don't have to reverse engineer the reasoning later. Or maybe we can create an issue?

I guess it's fallout from the retained render world?

Reply:
I just removed the comment. I don't actually remember why I wrote it, embarrassingly enough.